managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3300,6 +3305,13 @@ private void scheduleTimeoutTask() {
                 checkReadTimeout();
             }), timeoutSec, timeoutSec, TimeUnit.SECONDS);
         }
+
+        if (config.getMaximumRolloverTimeMs() > 0) {
```

Comment:
I think it would be better to separate the timing task of checking that current ledger is full from `scheduleTimeoutTask` and put it in the ledger `initialize` method.

Reply:
Ok, I move it into individual method.